.github/workflows/test.yml

```diff
@@ -232,7 +232,7 @@ jobs:
     uses: ./.github/workflows/test_datasets.yml
 
   test-full:
-    needs: test-nosoftdeps
+    needs: [test-base, test-module, test-other]
```

Comment:
Doubt: what exactly does this change do? Does it mean that now test-full and test-unix-pandas1 will run (entirely) if and only if all of the following 3 is true?

- test-base runs and completes successfully (as opposed to being skipped after detect step)
- test-module runs and completes successfully (as opposed to being skipped after detect step)
- test-other runs and completes successfully (as opposed to being skipped after detect step)

For all other cases (I don't know how common is that), it will be skipped. Is that correct understanding? Is it basically same as disabling old CI?

Reply:
hm, I see - I was thinking of, all three either complete or are skipped.

Is there a way to make these jobs appear as succeeded if they were skipped?

Reply:
https://github.com/orgs/community/discussions/45058

Will this help?